WebJobs.sln

```diff
@@ -51,16 +47,12 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TestProjects", "TestProject
 EndProject
 Project("{6EC3EE1D-3C4E-46DD-8F32-0CC8E7565705}") = "FSharpFunctions", "test\TestProjects\FSharpFunctions\FSharpFunctions.fsproj", "{11702A4B-8402-4082-BE38-4F0C2CBBF61D}"
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Benchmarks", "test\Benchmarks\Benchmarks.csproj", "{78F8086D-8313-477A-B24F-E475A690880A}"
```

Comment:
Do these need to be ported to use t2 extensions? These tests seem to only be testing json serialization.

Reply:
Just remove the Storage.Extensions dependency from the project as well as the WebjobsExtensionsStorage test file, leaving the project for future benchmarks. The existing extension test relies on internals visible access which we don't have for Track2. If we want to re-add these tests, that'll happen in the Track2 repo.